plugins/reporters/cyclonedx/src/funTest/kotlin/CycloneDxReporterFunTest.kt

```diff
@@ -99,6 +100,18 @@ class CycloneDxReporterFunTest : WordSpec({
             }
         }
 
+        "the expected XML file even if some copyrights contain non printable characters" {
+            val jsonOptions = optionSingle + mapOf("output.file.formats" to "xml")
+            val bomFileResults = generateReport(ORT_RESULT_WITH_ILLEGAL_COPYRIGHTS, jsonOptions)
```

Comment:
To maintain the AAA pattern from other tsets, there should be an empty line before this line which performs the actual action.